src/cc/export/helpers.h

```diff
@@ -410,5 +410,27 @@ int bpf_num_cpus() asm("llvm.bpf.extra");
 
 #define lock_xadd(ptr, val) ((void)__sync_fetch_and_add(ptr, val))
 
+#ifdef __powerpc__
+#define PT_REGS_PARM1	gpr[3]
+#define PT_REGS_PARM2	gpr[4]
+#define PT_REGS_PARM3	gpr[5]
+#define PT_REGS_PARM4	gpr[6]
+#define PT_REGS_PARM5	gpr[7]
+#define PT_REGS_PARM6	gpr[8]
+#define PT_REGS_RC	gpr[3]
+#define PT_REGS_IP	nip
+#define PT_REGS_SP	sp
+#else
```

Comment:
Wouldn't it be cleaner to have an else at the end so that any unsupported architectures are caught with a saner error message?

In either case, I will see about adding a top-level supported architecture check to prevent the compiler trying too hard before getting this far.